README.rst

```diff
@@ -34,8 +34,19 @@ Files this package must make available
 ``/logo-trademark.svg`` A variant of the logo with a trademark ® or ™. Note: This file must be present. If you don't have a trademark variant of your logo, copy your regular logo and use that.
 
 .. image:: /logo-trademark.svg
-    :alt: edX
+    :alt: logo
+    :width: 128px
+
+``/logo-white.svg`` A variant of the logo for use on dark backgrounds
+
+.. image:: /logo-white.svg
+    :alt: logo
     :width: 128px
 
+``/favicon.ico`` A site favicon
+
+.. image:: /favicon.ico
+    :alt: favicon
+    :width: 32px
```

Comment:
nit: Not sure it matters all that much, but the width used for the favicon.ico here differs from the favicon width used in https://github.com/edx/brand-edx.org/pull/13.